### ISSUE TITLE
Queen stomps are way too quiet

### ISSUE THREAD

**Issue by Skarletto** [RMC-14/RMC-14#4716]
Legit cannot hear the queen till she's in front of me yet I hear everything else perfectly fine. people racking their gun is 10x louder than the queen

one of the most vital sounds in the game

**Comment by Kadeo64** [comment]
IMO, vital sounds like queen stomps and (your) telephone ringing should be made into a separate sound group (like how lobby music/midi/ambience are separate groups) in settings so you can adjust it independently of the other sound effects

probably would be hell to code though

**Comment by deltanedas** [comment]
armourless marine that has all game sounds muted except for queen stomp :trollface:

**Comment by Kadeo64** [comment]
> armourless marine that has all game sounds muted except for queen stomp :trollface:

let em do it imo
they miss resin doors opening and other telltale audio signs of xeno activity (fruits being planted and eggs opening)
